cypress/support/testcase.js

```diff
@@ -81,8 +81,9 @@ export function reviewTestCase({ otherProduct }, user = false) {
   const title = user ? 'signed in' : 'anonymous'
 
   it(`Validate user added comments shown to this ${title} user`, () => {
```

Comment:
On first testcase, we are verifying for the anonymous user so, you should update this function
So, that it can show right title

https://github.com/vtex-apps/reviews-and-ratings/runs/6188318970?check_suite_focus=true - Currently it shows like this "Validate user added comments shown to this anonymous user" it should be "Validate **anonymous user** added comments shown to this anonymous user"


your function needs to decide how to show it like Validate user / Validate anoymous user